templates/containers/testimonials.html

```diff
@@ -0,0 +1,169 @@
+{% extends "containers/base_containers.html" %}
+
+{% block title %}Docker Engine on Ubuntu | Containers{% endblock %}
+
+{% block meta_description %}Canonical works in partnership with Docker Inc. to provide customers with a single path for the commercial support of Docker Engine running in production Ubuntu environments.{% endblock meta_description %}
+
+{% block meta_keywords %}Docker, Docker Engine, Docker Kubernetes, containers, Ubuntu OpenStack, Canonical, The Canonical Distribution of Kubernetes, OpenStack, cloud, Juju, Charms{% endblock meta_keywords %}
+
+{% block extra_body_class %}containers-home{% endblock %}
+
+{% block second_level_nav_items %}
+<div class="strip-inner-wrapper">
+  {% include "templates/_nav_breadcrumb.html" with section_title="Containers" page_title="Testimonials" %}
+</div>
+{% endblock second_level_nav_items %}
+
+{% block content %}
+<section class="row p-testimonial row-grey">
```

Comment:
Fixed also are quotation marks wrapping at the end of text, I’ll add an issue to the Vanilla pattern as it’s evident there too